ui/app/mainui/activitycenter/views/ActivityNotificationCommunityTokenReceived.qml

```diff
@@ -6,65 +6,108 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
 
-import shared 1.0
-import shared.panels 1.0
 import utils 1.0
 
-import "../panels"
-import "../popups"
-import "../stores"
+import AppLayouts.Wallet 1.0
 
 ActivityNotificationBase {
     id: root
 
+    // Community properties:
+    required property string communityId
     required property string communityName
     required property string communityImage
-    required property string communityColor
+
+    // Notification type related properties:
+    property bool isFirstTokenReceived: root.notification.isFirstTokenReceived
+    property bool isAssetType: root.notification.tokenType === Constants.TokenType.ERC20
+
+    // Token related properties:
+    property string tokenAmount: root.notification.tokenAmount
+    property string tokenName: root.notification.tokenName
+    property string tokenSymbol: root.notification.tokenSymbol
+    property string tokenImage: root.notification.tokenImage
+
+    // Wallet related:
+    property string walletAccountName: root.notification.walletAccountName
+    property string txHash: root.notification.txHash
```

Comment:
These should be read-only? or are they writable b/c of storybook?

Reply:
Yeah, they are writable bc of `storybook`!